File: Python/1_Iniciante/1044.py
ipt = [int(i) for i in input().split()]
ipt.append(ipt[0])

for i in range (len(ipt)):
    div = ipt[i]/ipt[(i+1)]
    if ((div - round(div)) == 0):
        print('Sao Multiplos')
        break
    elif(i==1):
        print('Nao sao Multiplos')
        break

File: Ruby/1_Iniciante/1044.rb
ipt = gets.split().map(&:to_i)
ipt.append(ipt[0])

ipt.length().times do |i|
    div = ipt[i].to_f/ipt[(i+1)]
    if (div - div.round()) == 0
        puts "Sao Multiplos"
        break
    elsif i == 1
        puts "Nao sao Multiplos"
        break
    end
end
